nextjs/src/lib/api.ts

```diff
@@ -11,7 +11,7 @@ export async function apiRequest(
     const response = await axios({ method, url, data });
 
     // verify request had 2xx status code
-    if (response.status !== 200) {
+    if (!(response.status >= 200 && response.status <= 299)) {
```

Comment:
taka a second dynamic look

Reply:
This change was necessary because the less flexible code would kick 201 and 204 responses and the like to the error handler.